spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -418,15 +411,178 @@ as part of an individual deployment request prefixed by the `app.<name of applic
+		new SpringApplication(DataFlowServer.class).run(args);
+	}
+}
+----
+
+* Add  `spring-cloud-services-starter-config-client` dependency in `pom.xml`. A maven example follows.
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-dataflow-server-cloudfoundry-autoconfig</artifactId>
+	<version>VERSION</version>
+</dependency>
+<dependency>
+    <groupId>io.pivotal.spring.cloud</groupId>
+    <artifactId>spring-cloud-services-starter-config-client</artifactId>
+    <version>1.4.0.RELEASE</version>
+</dependency>
+----
+
+* Build the application locally
+
+This completes the custom build of Spring Cloud Data Flow with `spring-cloud-services-starter-config-client`
+library included as the dependency.
+
+The final über-jar is now ready to be deployed to Cloud Foundry. With this setup and having the deployed application
+bound to config-server service on Cloud Foundry, we can successfully negotiate, read, and resolve centralized properties
+at the runtime.
 
 Follow the http://docs.pivotal.io/spring-cloud-services/config-server[documentation]
-for Config Server for Pivotal Cloud Foundry.
-For more details, please refer to Spring Cloud Services
+for Config Server for Pivotal Cloud Foundry. For more details, please refer to Spring Cloud Services
 http://docs.pivotal.io/spring-cloud-services/client-dependencies.html#config-server[client-dependencies documentation].
 
+==== Stream, Task, and Spring Cloud Config Server
+Similar to Spring Cloud Data Flow server, it is also possible to configure both the stream and task applications to
+resolve the centralized properties from config-server.
+
+Let's assume you'd like to read properties for `time-source` application from config-server.
+
+* Download the `time-source` application starter with "Rabbit binder starter" from http://start-scs.cfapps.io/
+* Load the downloaded project in an IDE
+* Add `@Import(org.springframework.cloud.stream.app.time.source.TimeSourceConfiguration.class)` to import the time-source's
+configuration properties
+* Add  `spring-cloud-services-starter-config-client` dependency
+
+* Build the application locally
+
+This completes the custom build of `time-source` application with `spring-cloud-services-starter-config-client`
+library included as the dependency. A maven example follows.
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.springframework.cloud.stream.app</groupId>
+  <artifactId>spring-cloud-starter-stream-source-time</artifactId>
+</dependency>
+<dependency>
+  <groupId>org.springframework.cloud</groupId>
+  <artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+</dependency>
+<dependency>
+  <groupId>io.pivotal.spring.cloud</groupId>
+  <artifactId>spring-cloud-services-starter-config-client</artifactId>
+  <version>1.4.0.RELEASE</version>
+</dependency>
+----
+
+The final `time-source` über-jar is now ready to be registered in Spring Cloud Data Flow. With this setup and having the
```

Comment:
Now great, you have your uberjar. I was kinda left hanging by how user should register that app because you either have it in your local maven repo or as a file in your local build dir. I don't see any docs talking about options and what is recommended way(if we have any recommendations).